### ISSUE TITLE
Mark datasets that must be included in CLARIAH Dataset Registry

### ISSUE THREAD

**Issue by ddeboer** [netwerk-digitaal-erfgoed/dataset-register#483]
How can we annotate dataset descriptions in the NDE Registry to make it clear they should be harvested by CLARIAH?

See https://github.com/CLARIAH/clariah-plus/issues/97.

**Comment by EnnoMeijers** [comment]
Before that we have to decide which party makes the decision that datasets should be marked for harvest by CLARIAH or which scenarios we need to support. I see three possibilities: 
- CLARIAH decides which datasets is of interest to them
- institutes promote their dataset to be of interest for the CLARIAH user group
- the Datasetregister team decides which group of datasets could be relevant for use within CLARIAH (in close coordination with CLARIAH)

**Comment by coret** [comment]
_- CLARIAH decides which datasets is of interest to them_

And administer these markings in the CLARIAH infrastructure? 

_- institutes promote their dataset to be of interest for the CLARIAH user group_

This would mean the source is changed, directed by our adjusted requirements (besides the includeInEuropeana?). Although I like this "at the source" option, I do wonder how many institutions will use an includeInClariah-like predicate. And, can a dataset supplier decide that it is a relevant CLARIAH dataset, or will the promotion be handled as a suggestion by CLARIAH?

_- the Datasetregister team decides which group of datasets could be relevant for use within CLARIAH (in close coordination with CLARIAH)_

I do not like the extra **manual** work these annotations would mean for the team. And bear in mind, you'd have to 'judge' every new (set of) datasets. NB: the same of course true for the first option (CLARIAH).

**Comment by ddeboer** [comment]
@coret and I will think about a way to describe this in the dataset description RDF and requirements.

**Comment by coret** [comment]
Implementation strategy:
- document in requirements > audience per dataset (not per datacatalog or per organisation)
- make separate "audience" graph where the Datasetregister Team stores initial audience triples for KB/B&G datasets for Clariah
- inform audiences (=aggregators) about this selection method for their aggregation purposes
- invite dataset providers to use `schema:audience` and phase out the "audience" graph (=shift of respondibility)

**Comment by ddeboer** [comment]
We have decided for now to publish only KB, B&G and IISG because these are CLARIAH partners.

**Comment by EnnoMeijers** [comment]
Indeed, currently all the datasets are being harvested, no filtering is done.
